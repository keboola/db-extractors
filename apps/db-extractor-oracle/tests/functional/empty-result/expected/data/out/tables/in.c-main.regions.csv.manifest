{
    "destination": "in.c-main.regions",
    "incremental": false
}
