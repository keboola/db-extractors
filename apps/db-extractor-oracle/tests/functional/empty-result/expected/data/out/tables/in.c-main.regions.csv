"REGION_ID","REGION_NAME"
